### PR TITLE
flintrock 0.11.0

### PR DESCRIPTION
**Pull request by chenrui333**

### CODE REVIEW

**Comment by zbeekman** [comment]
The inreplace for pyaml is failing. I suspect some resources need updating too.

**Review by zbeekman — CHANGES_REQUESTED** [review]
Please get rid of the `inreplace` being applied to setup py for pyaml, and update resources according to https://github.com/nchammas/flintrock/blob/v0.11.0/setup.py#L52-L62

Thanks!

**Comment by chenrui333** [comment]
ack, resume

**Comment by chenrui333** [comment]
Should be good now.

File: Formula/flintrock.rb [hunk 1]
```diff
@@ -106,10 +106,12 @@ class Flintrock < Formula
     sha256 "70e8a77beed4562e7f14fe23a786b54f6296e34344c23bc42f07b15018ff98e9"
   end
 
-  def install
-    # Python 3.7 compat
-    inreplace "setup.py", "PyYAML == 3.12", "PyYAML == 3.13"
+  resource "urllib3" do
```
Comment (chenrui333) [inline Formula/flintrock.rb]: This is needed for test step.
```
==> /usr/local/Cellar/flintrock/0.11.0/bin/flintrock
Traceback (most recent call last):
  File "/usr/local/Cellar/flintrock/0.11.0/bin/flintrock", line 6, in <module>
    from pkg_resources import load_entry_point
  File "/usr/local/Cellar/flintrock/0.11.0/libexec/lib/python3.7/site-packages/pkg_resources/__init__.py", line 3241, in <module>
    @_call_aside
  File "/usr/local/Cellar/flintrock/0.11.0/libexec/lib/python3.7/site-packages/pkg_resources/__init__.py", line 3225, in _call_aside
    f(*args, **kwargs)
  File "/usr/local/Cellar/flintrock/0.11.0/libexec/lib/python3.7/site-packages/pkg_resources/__init__.py", line 3254, in _initialize_master_working_set
    working_set = WorkingSet._build_master()
  File "/usr/local/Cellar/flintrock/0.11.0/libexec/lib/python3.7/site-packages/pkg_resources/__init__.py", line 583, in _build_master
    ws.require(__requires__)
  File "/usr/local/Cellar/flintrock/0.11.0/libexec/lib/python3.7/site-packages/pkg_resources/__init__.py", line 900, in require
    needed = self.resolve(parse_requirements(requirements))
  File "/usr/local/Cellar/flintrock/0.11.0/libexec/lib/python3.7/site-packages/pkg_resources/__init__.py", line 786, in resolve
    raise DistributionNotFound(req, requirers)
pkg_resources.DistributionNotFound: The 'urllib3<1.25,>=1.20' distribution was not found and is required by botocore
```

**Comment by chenrui333** [comment]
@fxcoudert this can be merged. 😄

**Pull request closed without merging**